bin.src/genDefectList.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+import argparse
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.daf.persistence as dafPersist
+from lsst.log import Log
+import numpy as np
+import os
+import pfs.drp.stella as drpStella
+try:
+    import lsst.afw.display as afwDisplay
+except ImportError:
+    afwDisplay = None
```

Comment:
import fails if display_ds9 is not setup. I could add display_ds9 to obs_pfs.table and remove the try/except block